compiler/transforms/dispatch_kernels.py

```diff
@@ -129,3 +169,4 @@ class DispatchKernels(ModulePass):
     def apply(self, ctx: MLContext, op: builtin.ModuleOp) -> None:
         PatternRewriteWalker(DispatchElementwiseMult()).rewrite_module(op)
         PatternRewriteWalker(DispatchQMatMul()).rewrite_module(op)
+        PatternRewriteWalker(DispatchSnaxALU()).rewrite_module(op)
```

Comment:
The naming you introduce here is better, but now it is a bit inconsistent with the previous other passes.

I'm wondering if we should co-locate these dispatch passes with their accelerator implementations in the accelerators folder?

Reply:
tbh this entire pass needs a rethink and do-over. I hope we can engineer a better system once the accelerator templates start to take better form